webapp/src/components/link_tooltip/link_tooltip.jsx

```diff
@@ -82,30 +88,27 @@ export const LinkTooltip = ({href, connected, gitlabURL, show}) => {
     }, [connected, href, show]);
 
     const getIconElement = () => {
-        const iconProps = {
-            size: 'small',
-            verticalAlign: 'text-bottom',
-        };
-
         let color;
         let icon;
-        const {OPENED_COLOR, CLOSED_COLOR, MERGED_COLOR} = STATE_COLOR_MAP;
+        const {OPENED_COLOR, CLOSED_COLOR, MERGED_COLOR, ISSUE_CLOSED_COLOR} = STATE_COLOR_MAP;
         switch (data.type) {
         case LINK_TYPES.MERGE_REQUESTS:
-            color = OPENED_COLOR;
-            icon = <GitPullRequestIcon {...iconProps}/>;
+            icon = (
+                <GitLabMergeRequestIcon
+                    fill={OPENED_COLOR}
+                    width={TOOLTIP_ICON_DIMENSION}
+                    height={TOOLTIP_ICON_DIMENSION}
+                />
```

Comment:
How come we only need `width` and `height` for this icon, and not the ones below?

Reply:
@mickmister Replied here: https://github.com/mattermost/mattermost-plugin-gitlab/pull/457#discussion_r1489240806